master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -850,6 +852,27 @@ private[celeborn] class Master(
       logInfo(s"Offered extra $offerSlotsExtraSize slots for $shuffleKey")
     }
 
+    if (authEnabled) {
+      // Pass application registration information to the workers
+      val pbApplicationMeta = PbApplicationMeta.newBuilder()
+        .setAppId(requestSlots.applicationId)
+        .setSecret(secretRegistry.getSecretKey(requestSlots.applicationId))
+        .build()
+      val transportMessage =
+        new TransportMessage(MessageType.APPLICATION_META, pbApplicationMeta.toByteArray)
+
+      slots.keySet().asScala.foreach { worker =>
+        try {
+          logInfo(s"Sending app registration info to ${worker.host}:${worker.internalPort}")
+          internalRpcEnvInUse.setupEndpointRef(
+            RpcAddress.apply(worker.host, worker.internalPort),
+            RpcNameConstants.WORKER_INTERNAL_EP).send(transportMessage)
+        } catch {
+          case t: Throwable =>
+            logError(s"Send application meta info to workers failed!", t)
```

Comment:
Workers will reach out to master in case the appId is unknown, right ?
Given this, make it `logInfo` ?

Reply:
Done